### PR TITLE
Updated authorization specs and policy scope in tests…

### PR DESCRIPTION
**Pull request by jgunnink**
… for better clarity and understanding

### CODE REVIEW

File: spec/authorization/policies/application_policy_spec.rb [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 RSpec.describe ApplicationPolicy do
   subject { policy }
-  let(:policy) { ApplicationPolicy.new(user, target_user) }
-  let(:target_user) { user }
+  let(:policy) { ApplicationPolicy.new(user, unused_object) }
+  #target object is not used in the policy, however pundit requires two objects to be passed in
+  let(:unused_object) { "unsed object" }
```
Comment (jordanmaguire) [inline spec/authorization/policies/application_policy_spec.rb]: typo (unsed)

File: spec/authorization/policies/dashboard_policy_spec.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 RSpec.describe DashboardPolicy do
   subject { policy }
-  let(:policy) { DashboardPolicy.new(user, target_user) }
-  let(:target_user) { user }
+  let(:policy) { DashboardPolicy.new(user, unused_object) }
+  #target object is not used in the policy, however pundit requires two objects to be passed in
+  let(:unused_object) { "unsed object" }
```
Comment (jordanmaguire) [inline spec/authorization/policies/dashboard_policy_spec.rb]: typo (unsed)

**Comment by jordanmaguire** [comment]
Fix typos then good to go

**Comment by jgunnink** [comment]
Congrats, you passed my test. :+1:

File: spec/authorization/policies/application_policy_spec.rb [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 RSpec.describe ApplicationPolicy do
   subject { policy }
-  let(:policy) { ApplicationPolicy.new(user, target_user) }
-  let(:target_user) { user }
+  let(:policy) { ApplicationPolicy.new(user, unused_object) }
+  #target object is not used in the policy, however pundit requires two objects to be passed in
```
Comment (pzi) [inline spec/authorization/policies/application_policy_spec.rb]: [PP] Space after # / before comment copy

**Pull request merged**